04week/tomorrows-technology/index.html

```diff
@@ -1,14 +1,59 @@
 <!DOCTYPE html>
 <html>
+
 <head>
+  <meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
+  <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;" name="viewport">>
   <link rel="stylesheet" href="css/style-xs.css" media="(max-width: 767px)" />
   <link rel="stylesheet" href="css/style-s.css" media="(min-width: 768px) and (max-width: 991px)" />
-  <link rel="stylesheet" href="css/style-m.css" media="(mind-width: 992px)" />
+  <link rel="stylesheet" href="css/style-m.css" media="(min-width: 992px)" />
   <link rel="stylesheet" href="css/style.css" />
```

Comment:
Next time, make sure that this file is listed before the rest of your style-*.css files